front-end/src/App.scss

```diff
@@ -1 +1,5 @@
 @import "~bootstrap/scss/bootstrap";
+@import '~antd/dist/antd.css';
+@import './assets/scss/main.scss';
```

Comment:
where are these being used? Using Ant Design together with Bootstrap/Reactstrap might cause our UI to break or act weird but I'm not sure. Chris probably knows

Reply:
Ideally we stick to one CSS framework. Bootstrap should be able to do what ant does. Here's an example of the Grid system in Bootstrap: https://getbootstrap.com/docs/4.0/layout/grid/. Let me know if you have any questions about mapping over Ant  concepts to Bootstrap

Reply:
Thanks for the link. I'll stick with Boostrap going forward.